openhcl/underhill_core/src/options.rs

```diff
@@ -203,6 +208,8 @@ impl Options {
         let gdbstub = parse_legacy_env_bool("OPENHCL_GDBSTUB");
         let gdbstub_port = parse_legacy_env_number("OPENHCL_GDBSTUB_PORT")?.map(|x| x as u32);
         let nvme_keep_alive = parse_env_bool("OPENHCL_NVME_KEEP_ALIVE");
+        let test_configuration =
+            legacy_openhcl_env("OPENHCL_TEST_CONFIG").map(|x| x.to_string_lossy().into_owned());
```

Comment:
this is a new option, so there's no need to support any `legacy_*`-based name aliasing on it.